### ISSUE TITLE
 Which is the best loss function to use if my dataset belongs to multi class classification ? #6913 

### ISSUE THREAD

**Issue by vishnuvardhan58** [pjreddie/darknet#2321]
@AlexeyAB Yolov3 uses binary cross-entropy loss function during class prediction. This is perfect if the dataset belongs to multi-label classification. But my dataset belongs to multi-class classification. So will use of binary cross-entropy as a loss function on my dataset a problem? If so what will be the best loss function to use while making a class prediction?
